sugoi-api-core/src/main/java/fr/insee/sugoi/core/exceptions/EntityNotFoundException.java

```diff
@@ -13,6 +13,10 @@
 */
 package fr.insee.sugoi.core.exceptions;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
I am not fond of having http specific semantic in `core` can't we do it in another way ?